src/cli/commands/lint.ts

```diff
@@ -50,16 +44,31 @@ linting ./openapi.yaml
       char: 'v',
       description: 'increase verbosity',
     }),
+    ruleset: flagHelpers.string({
+      char: 'r',
+      description: 'path to a ruleset file (supports http)',
+    }),
   };
 
   public static args = [{ name: 'source' }];
 
   public async run() {
     const { args, flags } = this.parse(Lint);
+    const { ruleset } = flags;
+    let ruleCollection;
+
+    if (ruleset) {
+      try {
+        this.log(`Reading ruleset`);
+        ruleCollection = await readRuleset(ruleset, this);
```

Comment:
RuleCollection can just be rules we don’t need to introduce this new term. It’s just a dictionary of rules.

Reply:
It's not a new term ;) It's exactly how the type we return is called, no? I'm not attached to this name thought, I'm happy to change it if bothers you.
`export type RuleCollection = Dictionary<Rule, string>;`

Reply:
Generally we don't seem to be using Hungarian notation so it seems a bit out of place. It's not like we have rulesetDictionary and nameStr. 😅

Reply:
lol I didnt realize that it's actually a THING :D 
Glad to learn that it's called hungarian notation :) 

will change!